tests/bot/cogs/moderation/test_infractions.py

```diff
@@ -0,0 +1,54 @@
+import textwrap
+import unittest
+from unittest.mock import AsyncMock, Mock, patch
+
+from bot.cogs.moderation.infractions import Infractions
+from tests.helpers import MockBot, MockContext, MockGuild, MockMember, MockRole
+
+
+class ShorteningTests(unittest.IsolatedAsyncioTestCase):
```

Comment:
Let's call it Truncation, not Shortening. Do you mind changing all the occurrances in this file?